Scarb.toml

```diff
@@ -15,6 +14,6 @@ alexandria_encoding = { git = "https://github.com/keep-starknet-strange/alexandr
 alexandria_merkle_tree = { git = "https://github.com/keep-starknet-strange/alexandria.git", tag = "cairo-v2.6.0" }
 alexandria_data_structures = { git = "https://github.com/keep-starknet-strange/alexandria.git", tag = "cairo-v2.6.0" }
 snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.20.0" }
-openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", tag = "v0.13.0" }
-controller_auth = { path = "packages/contracts/auth" }
-controller_session = { path = "packages/contracts/session" }
+openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", tag = "v0.12.0" }
+argent = {git = "https://github.com/piniom/argent-contracts-starknet.git", branch = "lib-target"}
```

Comment:
Lets switch to the cartridge-gg fork for now, we can do it as a follow up if easier